### PR TITLE
EDA on Pneumonia Data Set 

### PR DESCRIPTION
**Pull request by bobbybendola**
Name: Steven Kuzhipala
Part of #28 

In this notebook, I have reviewed if there is any bias in the given data set for pneumonia, by checking using matplotlib the class sizes for both the classes. After doing so, there was a clear difference between the number of images available for the NORMAL class and less for the pneumonia class. That means that during the training process, the model could be biased in its classification as well, simply because of the data we train it with.

To solve this problem, I decided to try to augment the number of images in the pneumonia class along with images that are simply resized and are in the same color format as the initial data set, to not cause any other differences for identification. I made sure that the number of dimensions or channels in each of the images were 3, including R, G and B so that the images are in grayscale not causing a distinct change in one of the classes.

To perform the rescalign and sizing I used the keras model.

### CODE REVIEW

**Review by margoglvz — APPROVED** [review]
Hi @bobbybendola! Ty for your PR and I really love your description! I think you have a solid foundation and I recommend that you move to the next step of the model training process. One thing, just be wary that it is totally normal if after training your model and using your validation set you notice you might need more filters or preprocessing functions (totally part of the process). I am excited to see how your project goes and possibly see you present at the General Project Final Presentation! Please reach out if you have any questions :D
